src/core/Geometry.js

```diff
@@ -335,7 +335,8 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		}
 
 		this.computeFaceNormals();
-
+		this.mergeVertices();
+		
```

Comment:
There seems to be trailing spaces. Please remove them otherwise the linter won't be happy :(